media/css/protocol/components/_menu-item.scss

```diff
@@ -99,58 +108,69 @@ $icon-size-md: 32px;
                     height: $icon-size-sm;
                     margin: 0;
                     position: absolute;
-                    top: $spacing-md;
+                    top: $spacing-sm;
                     width: $icon-size-sm;
                 }
             }
         }
-
-        .c-menu-item-link,
-        .c-menu-item-head {
-            @include border-box;
-            height: 100%;
-            padding: $spacing-md;
-        }
-
-        .c-menu-item-desc {
-            @include text-body-xs;
-            display: block;
-            margin: $spacing-sm 0 0;
-        }
     }
 
     @media #{$mq-md} {
-        width: 200px;
+        width: 260px;
 
         .c-menu-item-title {
             @include text-title-xxs;
         }
-
-        // Hide descriptions in non-grid browsers, to shorten the menu
-        .c-menu-item-desc {
-            display: none;
-
-            @supports (display:grid) {
-                display: block;
-            }
-        }
     }
 
     @media #{$mq-lg} {
-        width: 260px;
-
         &.mzp-has-icon {
             .c-menu-item-head,
             .c-menu-item-link {
+                padding: $spacing-md;
                 @include bidi(((padding-left, $spacing-md + $icon-size-md + $spacing-md, padding-right, 0),));
 
                 .c-menu-item-icon {
                     @include bidi(((left, $spacing-md, right, auto),));
                     height: $icon-size-md;
                     width: $icon-size-md;
+                    top: $spacing-md;
                 }
             }
         }
+
+        .c-menu-item-desc {
+            display: block;
+            margin-top: $spacing-sm;
+        }
+    }
+}
+
+// Hide descriptions to shorten the menu when nav is sticky
+// Otherwise long menus can't be scrolled
+@media #{$mq-md} and #{$mq-tall} {
```

Comment:
`mq-tall` equates to `min-heght: 600px`, so if I have a shorter screen than this value, I still get the overflowing descriptions. Would it be better to use a `max-hight` value here instead?

Reply:
At >600px the nav isn't sticky so you can scroll even if it overflows. Kropp and I felt the descriptions are valuable enough content that we should keep them visible whenever possible. It's really the combination of narrow window + tall (hence sticky nav) that makes trouble.

But thinking about it more now, even if it's scrollable that doesn't mean it's easy to scroll while the menu is open (I'm a habitual scroll-wheel users but not everyone is). Once you move your mouse the menu closes and you can't get it back without scrolling back to the top. So for some small number of people who scroll in odd ways and have viewports just the right size, they may never be able to reach the bottom of the dropdown... so yeah, maybe this should be `$mq-short`.